app/src/main/java/com/example/sharingang/ReportFragment.kt

```diff
@@ -37,25 +37,17 @@ class ReportFragment : Fragment() {
         reportedId = args.reportedId
         reportedUsername = args.reportedName
 
-
-        val radioButtons = listOf(
-            binding.radioUsername, binding.radioProfilePicture, binding.radioItem, binding.radioOther
-        )
-        setupRadioButtons(radioButtons)
+        setupRadioButtons()
         setupCancelButton()
-        setupButtonOk(radioButtons)
-        binding.textReportedUsername.text = "Reporting $reportedUsername"
+        setupButtonOk()
+        "Reporting $reportedUsername".also { binding.textReportedUsername.text = it }
```

Comment:
Isn't there a way not to have the string hardcoded?

Reply:
I could move "Reporting" to strings.xml, but then I would still need to concatenate, so I'm not really sure about it. Do you prefer that I make a string resource for "Reporting" and then set the text to `getString(...) $reportedUsername`?